Describe commands causes panic for newly created db

```
./kubedb describe pg postgres-demo
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x13ace0f]

goroutine 1 [running]:
github.com/k8sdb/kubedb/pkg/cmd/describer.(*humanReadableDescriber).describePostgres.func1(0x1f5a8a0, 0xc42007ef00, 0xc420217f10, 0x0)
	/home/tamal/go/src/github.com/k8sdb/kubedb/pkg/cmd/describer/k8sdb_describer.go:142 +0x5ff
github.com/k8sdb/kubedb/pkg/cmd/describer.tabbedString(0xc420525408, 0x1f55160, 0xc420217a40, 0xc4200eec60, 0x0)
	/home/tamal/go/src/github.com/k8sdb/kubedb/pkg/cmd/describer/lib.go:16 +0xbc
github.com/k8sdb/kubedb/pkg/cmd/describer.(*humanReadableDescriber).describePostgres(0xc4203b1290, 0xc420482a80, 0xc42003cccf, 0x40fccf, 0xc4205894d0, 0xc420026400, 0x44faa0)
	/home/tamal/go/src/github.com/k8sdb/kubedb/pkg/cmd/describer/k8sdb_describer.go:151 +0x342
github.com/k8sdb/kubedb/pkg/cmd/describer.(*humanReadableDescriber).(github.com/k8sdb/kubedb/pkg/cmd/describer.describePostgres)-fm(0xc420482a80, 0xc42003cccf, 0x0, 0x0, 0x0, 0x0)
	/home/tamal/go/src/github.com/k8sdb/kubedb/pkg/cmd/describer/describer.go:47 +0x3e
reflect.Value.call(0x14dae60, 0xc4201b0ef0, 0x13, 0x1689728, 0x4, 0xc4205899f8, 0x2, 0x2, 0x1, 0x143c720, ...)
	/usr/local/go/src/reflect/value.go:434 +0x91f
reflect.Value.Call(0x14dae60, 0xc4201b0ef0, 0x13, 0xc4205899f8, 0x2, 0x2, 0xc420482a80, 0x0, 0x0)
	/usr/local/go/src/reflect/value.go:302 +0xa4
github.com/k8sdb/kubedb/pkg/cmd/describer.(*humanReadableDescriber).Describe(0xc4203b1290, 0x1f547e0, 0xc420482a80, 0xc42003cccf, 0x0, 0x0, 0x8, 0xc4202b3920)
	/home/tamal/go/src/github.com/k8sdb/kubedb/pkg/cmd/describer/describer.go:101 +0x2ee
github.com/k8sdb/kubedb/pkg/cmd.RunDescribe(0x1f72100, 0xc4202cc440, 0x1f5a4a0, 0xc42000c018, 0x1f5a4a0, 0xc42000c020, 0xc4201526c0, 0xc4202cc340, 0x2, 0x2, ...)
	/home/tamal/go/src/github.com/k8sdb/kubedb/pkg/cmd/describe.go:118 +0x7cb
github.com/k8sdb/kubedb/pkg/cmd.NewCmdDescribe.func1(0xc4201526c0, 0xc4202cc340, 0x2, 0x2)
	/home/tamal/go/src/github.com/k8sdb/kubedb/pkg/cmd/describe.go:47 +0xf7
github.com/k8sdb/kubedb/vendor/github.com/spf13/cobra.(*Command).execute(0xc4201526c0, 0xc4202cc220, 0x2, 0x2, 0xc4201526c0, 0xc4202cc220)
	/home/tamal/go/src/github.com/k8sdb/kubedb/vendor/github.com/spf13/cobra/command.go:651 +0x23a
github.com/k8sdb/kubedb/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420415d40, 0xc420395680, 0xc420394d80, 0xc42023d421)
	/home/tamal/go/src/github.com/k8sdb/kubedb/vendor/github.com/spf13/cobra/command.go:726 +0x339
github.com/k8sdb/kubedb/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420415d40, 0xc42000c010, 0x1f5a4a0)
	/home/tamal/go/src/github.com/k8sdb/kubedb/vendor/github.com/spf13/cobra/command.go:685 +0x2b
github.com/k8sdb/kubedb/cmd/kubedb/app.Run(0xc4200001a0, 0xc4200001a0)
	/home/tamal/go/src/github.com/k8sdb/kubedb/cmd/kubedb/app/kubedb.go:11 +0x70
main.main()
	/home/tamal/go/src/github.com/k8sdb/kubedb/cmd/kubedb/kubedb.go:10 +0x22

```

--- a/pkg/cmd/describer/k8sdb_describer.go
+++ b/pkg/cmd/describer/k8sdb_describer.go
@@ -3,7 +3,6 @@ package describer
 import (
 	"fmt"
 	"io"
-	"time"
 
 	"github.com/golang/glog"
 	tapi "github.com/k8sdb/apimachinery/api"
@@ -53,7 +52,7 @@ func (d *humanReadableDescriber) describeElastic(item *tapi.Elastic, describerSe
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", item.Name)
 		fmt.Fprintf(out, "Namespace:\t%s\n", item.Namespace)
-		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", item.CreationTimestamp.Time.Format(time.RFC1123Z))
+		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", timeToString(&item.CreationTimestamp))
 		if item.Labels != nil {
 			printLabelsMultiline(out, "Labels:", item.Labels)
 		}
@@ -119,7 +118,7 @@ func (d *humanReadableDescriber) describePostgres(item *tapi.Postgres, describer
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", item.Name)
 		fmt.Fprintf(out, "Namespace:\t%s\n", item.Namespace)
-		fmt.Fprintf(out, "StartTimestamp:\t%s\n", item.CreationTimestamp.Time.Format(time.RFC1123Z))
+		fmt.Fprintf(out, "StartTimestamp:\t%s\n", timeToString(&item.CreationTimestamp))
 		if item.Labels != nil {
 			printLabelsMultiline(out, "Labels:", item.Labels)
 		}
@@ -173,9 +172,9 @@ func (d *humanReadableDescriber) describeSnapshot(item *tapi.Snapshot, describer
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", item.Name)
 		fmt.Fprintf(out, "Namespace:\t%s\n", item.Namespace)
-		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", item.CreationTimestamp.Format(time.RFC1123Z))
+		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", timeToString(&item.CreationTimestamp))
 		if item.Status.CompletionTime != nil {
-			fmt.Fprintf(out, "CompletionTimestamp:\t%s\n", item.Status.CompletionTime.Format(time.RFC1123Z))
+			fmt.Fprintf(out, "CompletionTimestamp:\t%s\n", timeToString(item.Status.CompletionTime))
 		}
 		if item.Labels != nil {
 			printLabelsMultiline(out, "Labels:", item.Labels)
@@ -220,12 +219,12 @@ func (d *humanReadableDescriber) describeDeletedDatabase(item *tapi.DeletedDatab
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", item.Name)
 		fmt.Fprintf(out, "Namespace:\t%s\n", item.Namespace)
-		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", item.CreationTimestamp.Format(time.RFC1123Z))
+		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", timeToString(&item.CreationTimestamp))
 		if item.Status.DeletionTime != nil {
-			fmt.Fprintf(out, "DeletionTimestamp:\t%s\n", item.Status.DeletionTime.Format(time.RFC1123Z))
+			fmt.Fprintf(out, "DeletionTimestamp:\t%s\n", timeToString(item.Status.DeletionTime))
 		}
 		if item.Status.WipeOutTime != nil {
-			fmt.Fprintf(out, "WipeOutTimestamp:\t%s\n", item.Status.WipeOutTime.Format(time.RFC1123Z))
+			fmt.Fprintf(out, "WipeOutTimestamp:\t%s\n", timeToString(item.Status.WipeOutTime))
 		}
 		if item.Labels != nil {
 			printLabelsMultiline(out, "Labels:", item.Labels)

--- a/pkg/cmd/describer/kube_describer.go
+++ b/pkg/cmd/describer/kube_describer.go
@@ -6,7 +6,6 @@ import (
 	"io"
 	"sort"
 	"strings"
-	"time"
 
 	"github.com/k8sdb/kubedb/pkg/cmd/printer"
 	"github.com/k8sdb/kubedb/pkg/cmd/util"
@@ -43,7 +42,7 @@ func (d *humanReadableDescriber) describeStatefulSet(namespace, name string, out
 	fmt.Fprint(out, "StatefulSet:\t\n")
 	fmt.Fprintf(out, "  Name:\t%s\n", ps.Name)
 	fmt.Fprintf(out, "  Replicas:\t%d current / %d desired\n", ps.Status.Replicas, ps.Spec.Replicas)
-	fmt.Fprintf(out, "  CreationTimestamp:\t%s\n", ps.CreationTimestamp.Time.Format(time.RFC1123Z))
+	fmt.Fprintf(out, "  CreationTimestamp:\t%s\n", timeToString(&ps.CreationTimestamp))
 	fmt.Fprintf(out, "  Pods Status:\t%d Running / %d Waiting / %d Succeeded / %d Failed\n", running, waiting, succeeded, failed)
 }
 

